### DISCUSSION TITLE
Does TTX support wildcards?  Converting multiple files to XML

### DISCUSSION THREAD

**Discussion by Davexx** [fonttools/fonttools#3691] [General]
Hello,
I am quite a noob to coding as I come form the design world.  I am attempting to use TTX to isolate the name Table using **TTX -t name**.  I attempted to add ***.otf** to the end, but it seems TTX cannot handle wildcard variables.
So, is there a better way to extract the name table from a list of fonts so that I may search and replace some name variables, then merge the tables back into an .OTF output?

**Comment by justvanrossum** [comment]
To dump multiple fonts works for me:

    ttx -t name some/folder/*.ttf

**Reply by Davexx** [reply]
I am using Windows CMD line and this gave me a "file not found" error:
`ttx -t name c:\Users\Davexx\Downloads\TTX\*.otf`
I even tried the forward slash:
`ttx -t name c:/Users/Davexx/Downloads/TTX/*.otf`


Apologies, You may need to spell this one out to me...

**Reply by justvanrossum** [reply]
Maybe that doesn't work on Windows CMD, but I can't check right now :(

**Reply by Davexx** [reply]
Thank you for checking!

**Reply by anthrotype** [reply]
does this work? (I don't have a windows machine to try this on)

```
for %f in (some\folder\*.ttf) do ttx "%f"
```

credits to [Gemini](https://g.co/gemini/share/73692dc8ab52)

**Reply by Davexx** [reply]
anthrotype,
 Good old batch DOS commands to the rescue!  That worked!
Thanks,
Davexx

**Reply by Davexx** [reply]
I figured out a way to merge the tables all back into the font files!
I thought I would post it here to help anyone on a similar path.
`for %f  in ("*.otf") do ttx -m "%~nf".otf "%~nf".ttx`
This command works in the directory containing all your *.otf font files and corresponding .ttx files to be merged.
